### DISCUSSION TITLE
Overwhelmed...

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#147] [Discuss a Topic]
I'm sure next week we will all look back and laugh, but I have to say, I'm a little overwhelmed at the moment with the flood of knowledge bestowed on us today. If you are feeling the same, you are NOT ALONE! I'm super grateful to have these videos to refer back to and these issue tickets that are rolling in!!! Thank you!

**Comment by drteresavasquez** [comment]
@danicrosby thank you for being so open and honest! I appreciate that you're verbalizing your feelings and we will try and help you along the way ❤️
